main/src/main/java/cgeo/geocaching/utils/HideActionBarUtils.java

```diff
@@ -0,0 +1,38 @@
+package cgeo.geocaching.utils;
+
+import cgeo.geocaching.R;
+import cgeo.geocaching.activity.AbstractBottomNavigationActivity;
+import cgeo.geocaching.settings.Settings;
+import cgeo.geocaching.ui.ViewUtils;
+
+import android.app.Activity;
+import android.view.View;
+
+import androidx.annotation.IdRes;
+import androidx.annotation.NonNull;
+
+public class HideActionBarUtils {
+
+    private HideActionBarUtils() {
+        // utility class
+    }
+
+    public static void showActionBarSpacer(@NonNull final Activity activity, final boolean showSpacer) {
+        activity.findViewById(R.id.hideActionBarSpacer).setVisibility(showSpacer ? View.VISIBLE : View.GONE);
+        if (!Settings.getMapActionbarAutohide()) {
```

Comment:
Wouldn't it be easier to just not call `setStableLayout` if MapActionbarAutohide is disabled, instead of adding a spacer to all views?

Reply:
And even if the spacer is still necessary, it would need to be placed at a different location

Android 7 - Navigation rail behind actionbar:

![grafik](https://user-images.githubusercontent.com/64581222/232248984-fbafdd83-5086-4d87-9399-ef5c1ee499f9.png)

Reply:
> Wouldn't it be easier to just not call `setStableLayout` if MapActionbarAutohide is disabled, instead of adding a spacer to all views?

Good idea, I'd tested getting rid of `setStableLayout()` at all before, but that didn't work. Removing it for "tap to hide actionbar disabled" does work, though. PR is adapted.

> And even if the spacer is still necessary, it would need to be placed at a different location
> Android 7 - Navigation rail behind actionbar:

Does happen without the spacer as well. Will try to adapt this PR or create a separate for it later.